app/screens/channel_info/channel_info.js

```diff
@@ -402,6 +436,20 @@ export default class ChannelInfo extends PureComponent {
         return isDirectMessage || isGroupMessage;
     };
 
+    renderUnarchiveChannel = () => {
+        const {canUndeleteChannel} = this.props;
+        const {canUseUnarchiveFeature} = this.props;
+        if (!canUseUnarchiveFeature || !canUndeleteChannel) {
```

Comment:
if removing `canUseUnarchiveFeature` is removed, then this if statement needs fixing